client/src/app/budgets/otherBudgetsDetail/otherBudgetsDetail.component.html

```diff
@@ -2,25 +2,39 @@
     <h2>
         Budget Edit - {{budget.year}} - {{budget.user.firstName}} {{budget.user.lastName}}
     </h2>
+    <table>
```

Comment:
Give it some style, this is how it looks like:

![image](https://user-images.githubusercontent.com/7927175/78001774-7a916a80-7336-11ea-8853-4e0727b5c301.png)